.github/workflows/docker.yml

```diff
@@ -0,0 +1,84 @@
+on:
+  schedule:
+    - cron: '0 7 */7 * *' # This should run once a week
+
+name: Docker images
+
+jobs:
+  build-images:
+    name: Build & push Docker images
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        runtime:
+          - name: fdo-20-08
```

Comment:
Idea: Can the matrix be composed of a tag/[list of stuff to install], this way we can provide super custom images if needed in the future with very low effort

Reply:
Like something that has gnome + io.elementary baseapp for elementary's stuff

Reply:
It's a good idea, but IMO I think that we should first make sure that matrix supports an array of objects, before we jump into doing that

Reply:
The matrix of objects does work, we use it in gtk-rs like this 
```yaml

    strategy:
      matrix:
        rust:
          - stable
          - beta
          - nightly
          - "1.48.0"
        conf:
          - { name: "atk", features: "v2_34", nightly: "--all-features", test_sys: true }
          - { name: "cairo", features: "png,pdf,svg,ps,use_glib,v1_16,freetype,script,xcb,xlib,win32-surface", nightly: "--features 'png,pdf,svg,ps,use_glib,v1_16,freetype,script,xcb,xlib,win32-surface'", test_sys: true }
          - { name: "gdk", features: "v3_24", nightly: "--features v3_24", test_sys: true }
          - { name: "gdk-pixbuf", features: "v2_40", nightly: "--all-features", test_sys: true }
          - { name: "gdkx11", features: "v3_24", nightly: "--all-features", test_sys: true }
          - { name: "gio", features: "v2_66", nightly: "--all-features", test_sys: false }
          - { name: "glib", features: "v2_66", nightly: "--all-features", test_sys: false }
          - { name: "graphene", features: "v1_10", nightly: "", test_sys: false }
          - { name: "gtk", features: "v3_24_9", nightly: "--all-features", test_sys: false }
          - { name: "pango", features: "v1_46", nightly: "--all-features", test_sys: false }
          - { name: "pangocairo", features: "", nightly: "--all-features", test_sys: false }
 ```

Reply:
Uhm, then I'll do it too